.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Sure we can put this here but I usually would rather not put editor specifics in the repo.
Wouldn't you rather put this in a global gitignore on your computer?

Reply:
Makes it easier for others as well. There's not a reason at all for it to be checked in, no harm in having this line in the gitignore.

Reply:
I will accept this impurity :)